### PR TITLE
[Doc] Add datalake faq&update Configuration.md

### PR DESCRIPTION
**Pull request by amber-create**
Based on https://github.com/StarRocks/docs.zh-cn/pull/6189/files
Signed-off-by: amber-create <yangyanping@starrocks.com>

Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/administration/Configuration.md [hunk 1]
```diff
@@ -420,6 +420,9 @@ BE static parameters are as follows.
 
 | Configuration item | Default | Unit | Description |
 | -------------------------------------------------- | ------------------------------------------------------------ | ------ | ------------------------------------------------------------ |
+| hdfs_client_enable_hedged_read           | false         | Specifies whether to enable the hedged read feature. This parameter is supported from v3.1 onwards.                                    |
```
Comment (Smith-Cruise) [inline docs/administration/Configuration.md:423]: 3.0,the same as below

File: docs/administration/Configuration.md [hunk 2]
```diff
@@ -420,6 +420,9 @@ BE static parameters are as follows.
...
+| hdfs_client_hedged_read_threadpool_size  | 128           | Specifies the size of the Hedged Read thread pool on your client. The thread pool size limits the number of threads to dedicate to the running of hedged reads in your client. This parameter is supported from v3.1 onwards. It is equivalent to the `dfs.client.hedged.read.threadpool.size` parameter in the `hdfs-site.xml` file of your HDFS cluster. |
+| hdfs_client_hedged_read_threshold_millis | 2500          | Specifies the number of milliseconds to wait before starting up a hedged read. For example, you have set this parameter to `30`. In this situation, if a read from a block has not returned within 30 milliseconds, StarRocks immediately starts up a new read against a different block replica. This parameter is supported from v3.1 onwards. It is equivalent to the `dfs.client.hedged.read.threshold.millis` parameter in the `hdfs-site.xml` file of your HDFS cluster. |
```
Comment (Smith-Cruise) [inline docs/administration/Configuration.md:425]: **HDFS client** immediately starts up a new read

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by Smith-Cruise — APPROVED** [review]

File: docs/data_source/datalake_faq.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+### Solution
+
+You can use one of the following solutions to resolve this issue:
+
+- **[Recommended]** Enable the [data cache](../data_source/data_cache.md) feature, which eliminates the impact of slow HDFS nodes on queries by automatically caching the data from external storage systems to the BEs of your StarRocks cluster.
+- Enable the [Hedged Read](https://hadoop.apache.org/docs/r2.8.3/hadoop-project-dist/hadoop-common/release/2.4.0/RELEASENOTES.2.4.0.html) feature. With this feature enabled, if a read from a block is slow, StarRocks starts up a new read, which runs in parallel to the original read, to read against a different block replica. Whenever one of the two reads returns, the other read is cancelled. **The Hedged Read feature can help accelerate reads, but it also significantly increases heap memory consumption on Java virtual machines (JVMs). Therefore, if your physical machines provide a small memory capacity, we recommend that you do not enable the Hedged Read feature.**
+
+#### [Recommended] Data Cache
+
+参见 [Data Cache](../data_source/data_cache.md)。
```
Comment (evelynzhaojie) [inline docs/data_source/datalake_faq.md:42]: chinese

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by hellolilyliuyi — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by evelynzhaojie — APPROVED** [review]

**Comment by amber-create** [comment]
@mergify backport branch-3.0 branch-3.1

**Pull request merged**